.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do you think we should also have a `settings.json` and `extensions.json` to have the linter run by default in VS Code, or is that built-in?